flutter/android/build.gradle

```diff
@@ -56,6 +56,6 @@ android {
 }
 
 dependencies {
-    api 'io.sentry:sentry-android:5.0.1'
+    api 'io.sentry:sentry-android:5.1.0-beta.6'
```

Comment:
Do we want a stable Flutter version with a beta Android version, again?

Reply:
we're also in a state beta version on the Flutter SDK, in case we are ready to go GA, we can either patch the stable version of the Android SDK, or sync the GA of the Android SDK.

Reply:
btw its just beta due to new performance features, its not doing anything fancier, so its supposed to be as stable as it is